tests/bootstrap.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Predis-Toolkit
+ *
+ * Additional classes and functionality to extend Predis
+ *
+ * @copyright 2015 One Mighty Roar
+ * @link http://onemightyroar.com
+ */
+
+// Load our autoloader, and add our Test class namespace
+$autoloader = require(__DIR__ . '/../vendor/autoload.php');
```

Comment:
Doesn't really matter, but you don't need this assignment anymore, since you're not using the variable. Could just be the require statement itself. :stuck_out_tongue: